package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --coverage --verbose",
```

Comment:
`jest --coverage --verbose --passWithNoTests` will allow ci to pass without tests, but we should remove once we add tests.

Reply:
thnx I didnt know about that flag, got basic tests to pass by adding 404 test